spk/radarr/src/service-setup.sh

```diff
@@ -35,12 +27,6 @@ service_postinst ()
     # If nessecary, add user also to the old group before removing it
     syno_user_add_to_legacy_group "${EFF_USER}" "${USER}" "${LEGACY_GROUP}"
     syno_user_add_to_legacy_group "${EFF_USER}" "${USER}" "users"
-
-    # Discard legacy obsolete busybox user account
-    BIN=${SYNOPKG_PKGDEST}/bin
-    $BIN/busybox --install $BIN >> ${INST_LOG}
-    $BIN/delgroup "${USER}" "users" >> ${INST_LOG}
-    $BIN/deluser "${USER}" >> ${INST_LOG}
```

Comment:
Is this safe to remove? It's very old and radarr doesn't depend on busybox itself

Reply:
Yes, we are removing them as we update packages